lib/internal/Magento/Framework/View/Asset/File.php

```diff
@@ -190,24 +191,23 @@ public function getContent()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function getFilePath()
     {
         return $this->filePath;
     }
 
     /**
-     * {@inheritdoc}
-     * @return File\Context
+     * @inheritdoc
      */
     public function getContext()
     {
         return $this->context;
     }
 
```

Comment:
@ihor-sviziev just clean DocBlock in core framework. To me previously return seem incorrect type